docs/abandoned-projects.rst

```diff
@@ -0,0 +1,105 @@
+Policy for Abandoned Projects
+=============================
+
+This policy describes the process by which a RTD project name may be changed.
+
+
+Rationale
+---------
+
+Conflict between the current use of the name and a different suggested use of
+the same name occasionally arise.  This document aims to provide general
+guidelines for solving the most typical cases of such conflicts.
+
+
+Specification
+-------------
+
+The main idea behind this policy is that RTD serves the community.  Every
+user is invited to upload content under the Terms of Use, understanding that it
+is at the sole risk of the user.
+
+While RTD is not a backup service, the maintainers do their best to keep that
+content accessible indefinitely in its published form.  However, in certain
+edge cases the greater community's needs might outweigh the individual's
+expectation of ownership of a project name.
+
+The use cases covered by this policy are:
+
+* Abandoned projects:
+
+    * renaming a project so that the original project name can be used by a
+      different project.
+
+* Active projects:
+
+    * resolving disputes over a name.
+
+
+Implementation
+--------------
+
+Reachability
+~~~~~~~~~~~~
+
+The user of RTD is solely responsible for being reachable by the maintainers
+for matters concerning projects that the user owns.  In every case where
+contacting the user is necessary, the maintainers will try to do so at least
+three times, using the following means of contact:
+
+* e-mail address on file in the user's profile;
+* e-mail addresses found in the given project's documentation; and
+* e-mail address on the project's home page.
+
+The maintainers will stop trying to reach the user after six weeks and the user
+will be considered *unreachable*.
+
+
+Abandoned projects
+~~~~~~~~~~~~~~~~~~
+
+A project is considered *abandoned* when ALL of the following are met:
+
+* owner is unreachable (see Reachability above);
+* no releases within the past twelve months; and
+* no activity from the owner on the project's home page (or no home page
+  found).
+
+All other projects are considered *active*.
+
+
+Renaming of an abandoned project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Projects are never renamed solely on the basis of abandonment.
+
+An *abandoned* project can be renamed (by appending "-abandoned" and a
+uniquifying integer if needed) for purposes of reusing the name when ALL of the
+following are met:
+
+* the project has been determined *abandoned* by the rules described above;
+* the candidate is able to demonstrate their own failed attempts to contact the
+  existing owner;
+* the candidate is able to demonstrate that the project suggested to reuse the
+  name already exists and meets notability requirements;
+* the candidate is able to demonstrate why a fork under a different name is not
```

Comment:
This seems a little draconian.

Reply:
It seems so to me too, but the request was for a policy based on PEP-0541 and this mirrors it exactly.

I especially dislike the bit about the candidate has to try to contact the project owner again after the system maintainers have already tried and failed (as part of designating the project as abandoned).

Reply:
Mmm.. yes, and this is conplicated ot demostrate in an objective way.

For example, as @ericholscher suggested before `python-manuel` as a name... but then, why/how do you demostrate that is not a valid name for your project? In this particular case, I think it's bad idea to deny just the `manuel` name to a project called `manuel` when the one occupying the name is *abandoned*. But that's also subjective... I think.